src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpFinancialController.cs

```diff
@@ -282,5 +286,22 @@ private static List<FinancialEvidence> GetFinancialEvidence(QnA.Api.Types.Sequen
                     return null;
             }
         }
+
+        private static string GetApplicableFinancialReviewStatus(string financialGrade)
+        {
+            switch (financialGrade)
+            {
+                case FinancialApplicationSelectedGrade.Outstanding:
+                case FinancialApplicationSelectedGrade.Good:
+                case FinancialApplicationSelectedGrade.Satisfactory:
+                    return FinancialReviewStatus.Approved;
+
+                case FinancialApplicationSelectedGrade.Exempt:
+                    return FinancialReviewStatus.Exempt;
+
+                default:
+                    return FinancialReviewStatus.Rejected;
+            }
+        }
```

Comment:
This should be done in the Handler within the API, not here

Reply:
Done